app-benchmarks/sysbench/sysbench-1.0.12.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools pax-utils
+
+DESCRIPTION="System performance benchmark"
+HOMEPAGE="https://github.com/akopytov/sysbench"
+SRC_URI="https://github.com/akopytov/sysbench/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="aio mysql postgres test"
+
+RDEPEND="aio? ( dev-libs/libaio )
+	mysql? ( virtual/libmysqlclient )
+	postgres? ( dev-db/postgresql:= )"
+DEPEND="${RDEPEND}
+	dev-lang/luajit:=
+	dev-libs/concurrencykit
+	dev-libs/libxslt
+	sys-devel/libtool:=
+	virtual/pkgconfig
+	test? ( dev-util/cram )"
+
+PATCHES=(
+	"${FILESDIR}/${P}-htmldir-fix.patch"
+)
+
+src_prepare() {
+	default
+
+	# remove bundled libs
+	rm -r third_party/luajit/luajit third_party/concurrency_kit/ck third_party/cram || die
+
+	eautoreconf
+}
+
+src_configure() {
+	local myeconfargs=(
+		$(use_enable aio aio)
+		$(use_with mysql mysql)
+		$(use_with postgres pgsql)
+		--without-attachsql
+		--without-drizzle
+		--without-oracle
+		--with-system-luajit
+		--with-system-ck
+	)
+
+	econf "${myeconfargs[@]}"
+}
+
+src_install() {
+	default
+
+	pax-mark m "${D}/usr/bin/${PN}"
+}
+
+src_test() {
+	emake check test
+}
```

Comment:
1. needs to go before `src_install`
2. Having a look at https://github.com/akopytov/sysbench/blob/master/tests/Makefile.am seems to suggest that `make test` runs the exact same stuff as `make check`, hence try obliterating `src_test` completely and see whether the defaults work

Reply:
Ad 2: check does not run the tests, it's just a check on the actual tests, the test target runs the tests.